DesignSafe-CI/portal
# DES-2263: Project Metadata Changes

## Overview: ##

## PR Status: ##


## Related Jira tickets: ##

* [DES-2263](https://jira.tacc.utexas.edu/browse/DES-2263)

## Summary of Changes: ##
* Replaced project metadata in project/publication templates with a new directive
* Removed some data embedding to simplify the data being passed around

## Testing Steps: ##
1. Basically click around in all the projects and publication views and make sure everything in the project metadata loads properly.

## UI Photos:

## Notes: ##
Known issue: Co-PIs do not display for type Other in the Publication Preview window.  We will fix this later; it's a bug from an earlier PR, so it wasn't broken by this code rework.
